tasks/setup.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Ensure docker dependencies are installed
```

Comment:
I guess this wasn't caught as all the places where we call this role already had these deps installed? Nice catch.

Reply:
I think it depends on how old the OS version is. After all, Molecule still uses `16.04`.